vhdbuilder/packer/components.json

```diff
@@ -67,11 +67,11 @@
       "multiArchVersions": [
         "v0.0.4.1",
         "v0.0.15",
-        "v0.1.1"
+        "v0.1.3"
       ],
       "prefetchOptimizations": [
         {
-          "version": "v0.0.15",
```

Comment:
we probably want to keep prefetching 0.0.15 if RP hasn't switched over to using 0.1.3 everywhere as of yet

Reply:
Once this merges in we want to change RP to 0.1.3. But if we need to have it in RP first I can make them separate PRs. 

Our current workstream for upgrading our images is AgentBaker before RP

Reply:
that's fine, let's keep prefetching 0.0.15 once the corresponding RP change has rolled out to all prod regions then